src/Squirrel/DownloadManager.cs

```diff
@@ -448,7 +448,7 @@ private long GetDownloadedFileSize(string fileUrl)
             string tempFilePath = "";
 
             // Attempts to download file part.
-            for (int i = 0; i < 3; i++)
+            for (int i = 0; i < 5; i++)
```

Comment:
Would not it be nice to define some CONNECTION_ATTEMPTS constant or config variable?

Reply:
Would be, agree, changed.